x/pylons/client/cli/tx_cookbook.go

```diff
@@ -52,21 +52,30 @@ pylonsd tx pylons create-cookbook "loud123456" "Legend of the Undead Dragon" "Co
 				return sdkerrors.Wrap(sdkerrors.ErrInvalidRequest, err.Error())
 			}
 
-			clientCtx, err := client.GetClientTxContext(cmd)
-			if err != nil {
-				return err
+			var clientCtx client.Context
+
+			altCtx := GetAltenativeContext()
+			if altCtx != nil {
```

Comment:
this if/else block has been used in multiple places, should be written as reusable in a function

Reply:
yeah, sure